src/components/clusterConfiguration/DiscoveryImageTypeControlGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Split, SplitItem } from '@patternfly/react-core';
+import { HelpIcon } from '@patternfly/react-icons';
+import { PopoverIcon } from '../ui';
+import { RadioField } from '../ui/formik/RadioField';
+
+const GROUP_NAME = 'imageType';
+
+const DiscoveryImageTypeControlGroupLabel = ({
+  text,
+  popoverContent,
+}: {
+  text: string;
+  popoverContent: string;
+}) => (
+  <>
+    {text}
+    <PopoverIcon
+      component={'a'}
+      variant={'plain'}
+      IconComponent={HelpIcon}
```

Comment:
I think we should unify on either `HelpIcon` or the `OutlinedQuestionCircleIcon` (no preference from my side).

Reply:
I agree. PF used HelpIcon in their examples. Also, I chose it because it made it look closer to the mock.